misc/de-metas-common/de-metas-common-util/src/main/java/de/metas/common/util/time/SystemTime.java

```diff
@@ -68,6 +73,11 @@ public void setFixedTimeSource(@NonNull final ZonedDateTime date)
 		setTimeSource(FixedTimeSource.ofZonedDateTime(date));
 	}
 
+	public void setFixedTimeSource(@NonNull final String zonedDateTime)
```

Comment:
i suggest you add a javadoc with something like "see {@link ZonedDateTimeparse(String)}"

Reply:
done